blocks/columns/columns.css

```diff
@@ -531,3 +547,56 @@ div.img-col > video {
     width: 469px;
   }
 }
+
+/* Text-image overlap View */
+.columns.text-and-image-in-column.overlap em a {
+  display: inline-flex;
+  width: 100%;
+  font-size: 1.25rem;
```

Comment:
I think the font sizes are also present in the `styles.css` file, please check for this and the other 2 `font-size` props below

Reply:
corrected it also, thanks